packages/apps-config/src/endpoints/testingRelayRococo.ts

```diff
@@ -584,6 +584,19 @@ export const testParasRococo: EndpointOption[] = [
       color: 'linear-gradient(180deg, rgba(32,90,172,1) 0%, rgba(26,72,138,1) 50%, rgba(13,36,69,1) 100%)',
       logo: nodesZeitgeistPNG
     }
+  },
+  {
+    info: 'giantTestnet',
+    paraId: 4227,
+    providers: {
+      GIANT: 'wss://rpc-1-us-east-1-testnetrococo.giantprotocol.org'
+    },
+    teleport: [-1],
+    text: 'GIANT Protocol',
```

Comment:
Just move this entry up to be alphabetical (based on this text). 

PS: This is the check CI fails on

Reply:
Done